tests/app/dao/test_tokens_dao.py

```diff
@@ -0,0 +1,45 @@
+import uuid
+from app.dao import tokens_dao
+
+from app.models import Token
+
+
+def test_should_create_token(notify_api, notify_db, notify_db_session, sample_service):
```

Comment:
A token is not passed to the create api endpoint, we generate a token and pass it back.